zulipterminal/ui_tools/buttons.py

```diff
@@ -220,13 +220,15 @@ def __init__(
             self.mark_muted()
 
     def mark_muted(self) -> None:
+        self.prefix_style = "muted"
```

Comment:
These are small feature changes, but you've added tests for the functions that you changed previously. Maybe move the tests across to the earlier commit, and adjust the tests for these small changes instead?

Reply:
Got it, I'll move them out :+1: